tests/conftest.py

```diff
@@ -1142,7 +1142,10 @@ def deterministic_asset_dataconnector_context(
     shutil.copy(
         file_relative_path(
             __file__,
-            "./test_fixtures/great_expectations_v013_no_datasource_stats_enabled.yml",
+            os.path.join(  # noqa: PTH118
```

Comment:
Please don't use os.path for this

Reply:
@Kilo59 It is an improvement of what was there before.  I can modernize it of course.

Reply:
I modernized it in a couple of places.